static/js/components/NewObservation.jsx

```diff
@@ -1,30 +1,50 @@
-import React from "react";
+// NewObservation.jsx
+import React, { useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
-// eslint-disable-next-line import/no-unresolved
 import Form from "@rjsf/mui";
 import validator from "@rjsf/validator-ajv8";
 import dataUriToBuffer from "data-uri-to-buffer";
 import { showNotification } from "baselayer/components/Notifications";
 import { fetchObservations, uploadObservations } from "../ducks/observations";
+import SearchableDropDown from "./SearchableDropDown";
 
 const NewObservation = () => {
   const { instrumentList } = useSelector((state) => state.instruments);
-  const { telescopeList } = useSelector((state) => state.telescopes);
```

Comment:
Look, its here. We grabbed the telescopeList to build that mapper and get the name of the telescope associated to each instrument, but you removed it.